web/packages/build/vite/tsconfigPaths.mjs

```diff
@@ -30,13 +30,6 @@ const rootDirectory = path.resolve(import.meta.dirname, '../../../..');
 
 export function tsconfigPathsPlugin() {
   return tsconfigPaths({
-    // Asking vite to crawl the root directory (by defining the `root` object, rather than `projects`) causes vite builds to fail
-    // with a:
-    //
-    // "Error: ENOTDIR: not a directory, scandir '/go/src/github.com/gravitational/teleport/docker/ansible/rdir/rdir/rdir'""
-    //
-    // on a Debian GNU/Linux 10 (buster) (buildbox-node) Docker image running on an arm64 Macbook macOS 14.1.2. It's not clear why
-    // this happens, however, defining the tsconfig file directly works around the issue.
-    projects: [path.resolve(rootDirectory, 'tsconfig.json')],
```

Comment:
This doesn't appear to be the case anymore. After clearing `node_modules` (because buildboxes use stuff from Docker's host dir) and replacing this with `root: rootDirectory`, I was able to successfully execute both `make -C build.assets teleterm` and `make -C build.assets webassets`.

---

Side note: I had to run `rm -rf .pnpm-store node_modules` a lot, because almost every execution of those Make targets would fail with an error related to symlinks:

```
! Corepack is about to download https://registry.npmjs.org/pnpm/-/pnpm-9.8.0.tgz
Scope: all 7 workspace projects
Lockfile is up to date, resolution step is skipped
Progress: resolved 1, reused 0, downloaded 0, added 0
Packages: +1410
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 1410, reused 0, downloaded 31, added 0
Progress: resolved 1410, reused 0, downloaded 95, added 86
Progress: resolved 1410, reused 0, downloaded 155, added 145
Progress: resolved 1410, reused 0, downloaded 184, added 178
Progress: resolved 1410, reused 0, downloaded 227, added 220
Progress: resolved 1410, reused 0, downloaded 277, added 270
Progress: resolved 1410, reused 0, downloaded 338, added 331
Progress: resolved 1410, reused 0, downloaded 378, added 369
Progress: resolved 1410, reused 0, downloaded 420, added 415
Progress: resolved 1410, reused 0, downloaded 473, added 471
Progress: resolved 1410, reused 0, downloaded 536, added 525
Progress: resolved 1410, reused 0, downloaded 564, added 557
Progress: resolved 1410, reused 0, downloaded 593, added 590
Progress: resolved 1410, reused 0, downloaded 614, added 609
 ERR_PNPM_Unknown system error -116  Unknown system error -116: Unknown system error -116, copyfile '/go/src/github.com/gravitational/teleport/.pnpm-store/v3/files/cf/83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e' -> '/go/src/github.com/gravitational/teleport/node_modules/.pnpm/resolve@2.0.0-next.5/node_modules/resolve_tmp_7/test/resolver/missing_main/index.js'
make: *** [teleterm] Error 1
```